.travis.yml

```diff
@@ -4,5 +4,6 @@ go:
     - 1.4
 
 before_script:
+    - go get -d -v ./...
```

Comment:
No problem. I can squash/fix while reviewing. All deps should be included in `Godeps/` so this should not be needed. Dockerclient has new dep `docker/pkg/units` that also needs to be included.